plugins/node/opentelemetry-instrumentation-hapi/src/instrumentation.ts

```diff
@@ -275,7 +271,6 @@ export class HapiInstrumentation extends InstrumentationBase {
     const oldHandler = plugin.register;
     const self = this;
     const newRegisterHandler = function (server: Hapi.Server, options: T) {
-      server.route;
```

Comment:
Do you understand why this statement was added in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1595 originally? I would *guess* it was left over debugging code, but I don't know.

Reply:
That's what I though. Since the tests are passing I assumed it was just a left-over. If not I would expect a comment here explaining why this is needed.